vidyut-lipi/src/extensions/vedic/yajurveda.rs

```diff
@@ -0,0 +1,46 @@
+//! Taittiriya Yajurveda sakha transliteration support.
+
+use super::{VedicAccent, VedicSakha};
+
+/// Taittiriya Yajurveda sakha extension.
+///
+/// Based on the Extended Baraha ASCII format used in udapaana:
+/// - `#` = udātta (high tone)
+/// - `q` = anudātta (low tone)
+/// - Special annotations like `(gm)`, `(gg)` for phonetic variations
+#[derive(Clone, Copy)]
+pub struct TaittiriyaYajurveda;
+
+impl VedicSakha for TaittiriyaYajurveda {
+    fn name(&self) -> &str {
+        "Taittiriya Yajurveda"
+    }
+    
+    fn accent_mappings(&self) -> Vec<(&'static str, &'static str, VedicAccent)> {
+        vec![
+            // Baraha format mappings (primary for Taittiriya)
+            ("#", "\u{0951}", VedicAccent::Udatta),      // udātta
+            ("q", "\u{0952}", VedicAccent::Anudatta),    // anudātta
+            ("=", "\u{0953}", VedicAccent::Svarita),     // svarita
+        ]
+    }
+    
+    fn phonetic_mappings(&self) -> Vec<(&'static str, &'static str)> {
+        vec![
+            // Special phonetic annotations from Taittiriya
+            ("(gm)", "ṁ"),      // Guttural + m sound
+            ("(gg)", "g̃"),      // Nasalized g
+            ("~M", "ṁ"),        // Special nasalization
```

Comment:
Nothing to do here, but note that for these chandrabindu nasals, there's a reduplication that happens. For example, `devaṁ eva` sounds like `devay(first y should be nasal)yeva`. We won't capture that here. That nasal should be indication enough how to pronounce that. Later on, we should have a more complex process for encoding the sound properly, but this is the right way to do this. From this we can produce that. Harder to go back, so our data shuld be as is.